cmake/autocmake_static_library.cmake

```diff
@@ -42,13 +43,13 @@ if(ENABLE_GENERIC)
     endif()
 
     if(DEFINED CMAKE_CXX_COMPILER_ID)
-        if(CMAKE_C_COMPILER_ID MATCHES GNU)
+        if(CMAKE_CXX_COMPILER_ID MATCHES GNU)
```

Comment:
@loriab Thanks for the heads up. It's already fixed in PCMSolver. I would suggest to remove the `autocmake_` prefix from the filename, since this file is not part of Autocmake.